packages/zwave-js/src/lib/commandclass/SceneControllerConfigurationCC.ts

```diff
@@ -386,7 +404,9 @@ export class SceneControllerConfigurationCCSet extends SceneControllerConfigurat
 			const groupCount = this.getGroupCountCached();
 			this.groupId = options.groupId;
 			this.sceneId = options.sceneId;
-			this.dimmingDuration = options.dimmingDuration;
+			// if dimmingDuration was missing, default to 0 seconds
+			this.dimmingDuration =
+				options.dimmingDuration ?? new Duration(0, "seconds");
```

Comment:
Shouldn't we use a `"default"` duration in this case?

Reply:
Yes, setting it to `0xFF` makes more sense, so `Duration(0, "default")`